### PR TITLE
Improved the ocaml.org/learn/portability.html page

### LINKED ISSUE #1336: Improve https://ocaml.org/learn/portability.html page
# Issue description

When you navigate to [learn/portability](https://ocaml.org/learn/portability.html) page, you will see a couple of tables which are quite difficult to read and comprehend. You can check the screenshots below.

image
image

I think it would make things a little easier to understand if we add padding  to the table rows, add table outline  and alternating rows are styled to become zebra striped. I believe it will even look better. For reference you can check [this W3C example](https://www.w3schools.com/howto/howto_css_table_zebra.asp).

### PR DESCRIPTION
**Pull request by mercybassey**
Improved the ocaml.org/learn/portability.html page

# Issue Description
The tables on the learn/portability page has a couple of tables which are quite difficult to read and comprehend and also not responsive on mobile devices.

Fixes #1336

## Changes Made

I made changes in the table, th, tr and td tags respectively in the bootstrap.css file
I made changes in the portability.md file by adding a div tag as a parent container to the second table on the learn/portability page  and inserted the second table in the div tag giving it a style of "overflow-x : auto" to make the table responsive on mobile devices. 

Before:

113113312-d4ab6100-9212-11eb-9553-f4ce92ec3254

113113362-e7259a80-9212-11eb-8356-57b594b87ba9

After:
Screenshot from 2021-04-08 00-00-07

Screenshot from 2021-04-08 00-01-28

and on mobile view:

mobileview

* **Please check if the PR fulfills these requirements**

- [ ] PR is descriptively titled and links the original issue above
- [ ] Before/after screenshots (if this is a layout change)
- [ ] Details of which platforms the change was tested on (if this is a browser-specific change)
- [ ] Context for what motivated the change (if this is a change to some content)

### CODE REVIEW

**Comment by mercybassey** [comment]
@avsm @patricoferris  I'd like you to review

File: site/css/bootstrap.css [hunk 1]
```diff
@@ -1728,19 +1728,37 @@ table {
...
-.table th,
+/* .table th,
 .table td {
   padding: 8px;
   line-height: 24px;
   text-align: center;
   vertical-align: center;
+} */
+
+/* Added these */
```
Comment (patricoferris) [inline site/css/bootstrap.css]: Can you remove this comment, I don't think it is providing any insight.

File: site/css/bootstrap.css [hunk 2]
```diff
@@ -1728,19 +1728,37 @@ table {
...
+th, td {
+  text-align: left;
+  padding: 8px;
 }
+/* Added these too */
```
Comment (patricoferris) [inline site/css/bootstrap.css]: Can you remove this one too.

File: site/css/bootstrap.css [hunk 3]
```diff
@@ -1728,19 +1728,37 @@ table {
...
+tr:nth-child(even){
+  background-color: #f2f2f2
+}
+
+/* Also added these */
```
Comment (patricoferris) [inline site/css/bootstrap.css]: And this one.

File: site/css/bootstrap.css [hunk 4]
```diff
@@ -1728,19 +1728,37 @@ table {
...
+  /* Added these */
+  border: 1px solid #ddd;
 }
 .table {
   width: 100%;
   margin-bottom: 24px;
   font-family: Lato, sans-serif;
 }
-.table th,
+/* .table th,
```
Comment (patricoferris) [inline site/css/bootstrap.css]: I believe you've commented this out, do you know what impact this has on any other table in the site (a quick look suggests there aren't any non OCaml manual ones, just wanted to know if you had checked).

**Review by patricoferris — COMMENTED** [review]
This is looking much more readable thanks! We can push it to staging soon to double check.

**Comment by mercybassey** [comment]
@patricoferris  thanks for your review i will remove the comments right away

File: site/css/bootstrap.css (hunk 4 above)
Comment (mercybassey) [reply site/css/bootstrap.css]: Checked and found no other tables on the Ocaml website expect the one on the learn/portability.html page. And everything looks fine.

**Pull request closed without merging**